src/lib/apiTypesV2/SubscriptionExpression.h

```diff
@@ -27,21 +27,30 @@
 */
 #include <string>
 
+#include "rest/StringFilter.h"
+
+
 
 /* ***********************************************
 *
-* SubscriptionExpression
+* SubscriptionExpression -
+*
+* NOTE
+*   This struct contains both 'q' and stringFilter.
+*   q is the 'plain string' of the stringFilter and it is used in parsing of V2 Subscriptions
+*   and when the q-string is read from the database.
 */
 struct SubscriptionExpression
 {
-  SubscriptionExpression(): isSet(false) {}
-
+SubscriptionExpression(): isSet(false) { }
```

Comment:
Not sure but... member methods in a class should be indent in 2 spaces, shouldn't they?